kolibri/core/discovery/utils/filesystem/base.py

```diff
@@ -0,0 +1,78 @@
+import logging
```

Comment:
`base` for me implies a different thing. It means that something is at the bottom of a hierarchy, i.e. everything depends on it. This seems different, rather it looks like it's the API that other code should use. Maybe instead of `base.py`, it should be called `api.py`, or perhaps import the most important functions here into `__init__.py`?